addons/slideshow/functions/fnc_createSlideshow.sqf

```diff
@@ -0,0 +1,75 @@
+/*
+ * Author: Jonpas, DaC
+ * Prepares necessary variables and default image.
+ *
+ * Arguments:
+ * 0: Objects <ARRAY>
+ * 1: Controller Objects <ARRAY>
+ * 2: Image Paths <ARRAY>
+ * 3: Action Names <ARRAY>
+ * 4: Slide Duration <NUMBER> (0 disables automatic transitions)
+ *
+ * Return Value:
+ * Parsed List <ARRAY>
+ *
+ * Example:
+ * [[object1, object2, object3], [controller1], ["images\image1.paa", "images\image2.paa"], ["Action1", "Action2"], 5] call ace_slideshow_fnc_createSlideshow;
+ *
+ * Public: Yes
+ */
+//#define DEBUG_MODE_FULL
+#include "script_component.hpp"
+
+PARAMS_5(_objects,_controllers,_images,_names,_duration);
+
+// Add controllers to objects if they support it
+{
+    if (typeOf _x in [CLASSNAMES_OBJECTS, CLASSNAMES_BOTH]) then {
+        _objects pushBack _x;
+    };
+} forEach _controllers;
+
+// Objects synced to the module
+{
+    if (typeOf _x in [CLASSNAMES_OBJECTS, CLASSNAMES_BOTH]) then {
```

Comment:
Read above, requires ACE interaction which is not enabled by default on 99% of the object that slideshow looks good on.